dluhc-component-library/src/components/siteSelectionForm/components/NumberInput.tsx

```diff
@@ -0,0 +1,26 @@
+interface NumberInputProps {
+  value: number;
+  onChange: (value: number) => void;
+}
+
+const NumberInput = ({ value, onChange }: NumberInputProps) => {

```

Comment:
Is there a reason we cant have one "Input" component that passes through any props we need? Is there a benefit to having them as two components?